### DISCUSSION TITLE
Biome js not showing as a suggested formatter option in vscode

### DISCUSSION THREAD

**Discussion by juliomuhlbauer** [biomejs/biome-vscode#23] [Q&A] [answered]
![image](https://github.com/biomejs/biome/assets/53458125/2226b8e8-f307-4cd2-8d3e-5d3f6b5c8c12)

**Comment by nhedger** [comment, marked as answer]
Hey @juliomuhlbauer,

Do you have a `biome.json` file at the root of your workspace?

If not, please create one and reload the VS Code window

**Comment by Mayan-t** [comment]
I think that this is the issue I have on all the dev PCs and all projects with Biome: it does not show Biome as options on files that are more than one folder deep from project root.
